File: atcoder.jp/contests/abc092/tasks/abc092_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const divCeil = (a: number, b: number) => Math.trunc((a + b - 1) / b);
const sum = (array: number[]) => array.reduce((a, b) => a + b, 0);

const main = () => {
  const N = +input();
  const [D, X] = input().split(" ").map(Number);
  const A = [...Array(N)].map(() => +input());

  const ans = X + sum(A.map(divCeil.bind(null, D)));
  console.log(ans);
};

main();

File: atcoder.jp/contests/abc092/tasks/abc092_b/main.py
from functools import partial


def div_ceil(a: int, b: int) -> int:
    return (a + b - 1) // b


N = int(input())
D, X = map(int, input().split())
A = [int(input()) for _ in range(N)]

ans = X + sum(map(partial(div_ceil, D), A))
print(ans)
